ui/packages/app/src/PrivateLayout.js

```diff
@@ -1,14 +1,14 @@
-import React from "react";
 import {
   ApplicationsContext,
   ApplicationsContextProvider,
   Header,
   PrivateRoute,
   ProjectsContextProvider
 } from "@caraml-dev/ui-lib";
-import config from "./config";
 import urlJoin from "proper-url-join";
+import React from "react";
 import { Outlet, useNavigate } from "react-router-dom";
+import config from "./config";
```

Comment:
why did the imports get re-sorted?

Reply:
Most likely because of my IDE's auto formatter.